modules/codegen/src/main/scala/com/twilio/guardrail/generators/SwaggerGenerator.scala

```diff
@@ -26,24 +26,48 @@ object SwaggerGenerator {
     }
 
     def apply[T](term: SwaggerTerm[L, T]): Target[T] = term match {
-      case ExtractOperations(paths, globalSecurityRequirements) =>
+      case ExtractCommonRequestBodies(components) =>
+        Target.pure(components.flatMap(c => Option(c.getRequestBodies)).fold(Map.empty[String, RequestBody])(_.asScala.toMap))
+
+      case ExtractOperations(paths, commonRequestBodies, globalSecurityRequirements) =>
         for {
           _ <- Target.log.debug("AkkaHttpServerGenerator", "server")(s"extractOperations(${paths})")
-          routes <- paths.traverse {
+          routes <- paths.traverse({
             case (pathStr, path) =>
               for {
-
                 operationMap <- Target.fromOption(Option(path.readOperationsMap()), "No operations defined")
-              } yield {
-                operationMap.asScala.toList.map {
+                operationRoutes <- operationMap.asScala.toList.traverse({
                   case (httpMethod, operation) =>
                     val securityRequirements = Option(operation.getSecurity)
                       .flatMap(SecurityRequirements(_, SecurityOptional(operation), SecurityRequirements.Local))
                       .orElse(globalSecurityRequirements)
-                    RouteMeta(pathStr, httpMethod, operation, securityRequirements)
-                }
-              }
-          }
+
+                    // For some reason the 'resolve' option on the openapi parser doesn't auto-resolve
+                    // requestBodies, so let's manually fix that up here.
+                    val updatedOperation = Option(operation.getRequestBody)
+                      .flatMap(rb => Option(rb.get$ref))
```

Comment:
Why does this need to be a consideration there?  Request bodies aren't like schemas; you can't really do anything but inline them, similar to parameters or headers.

Reply:
The fact that you can call `get$ref` would suggest that this is a `Schema[_]` though, no? If it's that, I would expect you could actually stick a literal there, even if they don't have examples for it

Reply:
No, it's not a `Schema[_]`.  Only stuff under `components/schemas` are `Schema[_]`s.  The rest are different object types; see https://github.com/swagger-api/swagger-core/blob/master/modules/swagger-models/src/main/java/io/swagger/v3/oas/models/Components.java#L41-L49

A `RequestBody` can _contain_ a `Schema[_]` under its `MediaType`, but this PR leaves it unresolved, allowing the regular machinery in Guardrail to resolve it.

Reply:
(As I mention in the PR description, the `resolve` option to the parser, which we use, auto-resolves `Parameter`, `Header`, and `Response` references.  It doesn't make sense to me that it doesn't also auto-resolve `ResponseBody` references too, since they're the same idea.  Especially if they auto-resolve `Response` references, since their structure and function is _very_ similar.)

Reply:
Ah, understood. I mistakenly thought that even `RequestBody` was `<: Schema[_]`, in which case we'd need to handle the other cases. Thanks!